examples/remote_procedure/toxicity_detection/README.md

```diff
@@ -0,0 +1,7 @@
+# Mephisto Toxicity Detection Demo
+
+This task presents the worker with an input to type in text. 
+
+The text can only be submitted if its toxicity is is calculated to be <= 0.5. 
+
+The toxicity is calculated from the detoxify python library.
```

Comment:
I think we can mention the `pip install detoxify` prereq step here

Reply:
Just pushed a change to the readme, should be fixed now